### DISCUSSION TITLE
Can't access the website

### DISCUSSION THREAD

**Discussion by RSeyda** [awawa-dev/HyperHDR#952] [General]
Hi!

I have not logged in to hyperhdr for about a month. I have tried searching around for any possible updates that could explain what is going on, but I haven't been able to find any.
I have the correct address bookmarked so I am sure that that isn't the issue. After waiting for a couple seconds I simply get a message that says that the site can't be reached. 
I have not fiddled with anything else ever since I first got it to work about a month ago.

I am thankful for any help I can get :)

**Comment by awawa-dev** [comment]
Hi
Is ambilight still working? I suspect a network problem. Check if the unit pings and if you can connect to it, e.g. via ssh if the service was enabled
